### ISSUE TITLE
Output vector class associations not consistent

### ISSUE THREAD

**Issue by theloni-monk** [ekzhang/fastseg#12]
Hi, I'm a bit confused on how to validate the model. The output of the MobilenetV3 model is Batch x 19 x ImgDims, but the labels range from 0-19, so there is exactly one missing class. It seems to me that taking argmax along the class dimension of the model output produces a result exactly one off from the true labels. Are we supposed to softmax the class dimension and assign the void label to the case where all probabilites are less than 0.5? 
If possible, please provide a script that generates outputs that can be readily compared to the labels via functional cross-entropy-loss, mean intersection-over-union, and absolute pixel accuracy.

**Comment by ekzhang** [comment]
The training and validation code is open-source https://github.com/ekzhang/semantic-segmentation
